bff/config.json

```
@@ -1,5 +1,7 @@
 {
   "port": "4000",
+  "time_dim_guard": 2,
+  "non_time_dim_guard":1,
```

Let's use the current limits:
```suggestion
  "time_dim_guard": 12,
  "non_time_dim_guard":3,
```